### ISSUE TITLE
Perception without tracking task

### ISSUE THREAD

**Issue by chenzyhust** [hustvl/VAD#36]
Why does Agent Query directly perform motion transformer without performing tracking tasks? Will this reduce performance?

**Comment by rb93dett** [comment]
Yes, VAD does not explicitly incorporate tracking. We found that using implicit features can actually achieve pretty good motion prediction results, and VAD primarily focuses on planning. Theoretically, incorporating tracking could further improve motion performance, but it requires a trade-off in terms of computational complexity and speed.
